src/state/slices/marketDataSlice/marketDataSlice.tsx

```diff
@@ -158,7 +158,7 @@ export const selectPriceHistoryByAssetTimeframe = createSelector(
     (priceHistory[timeframe][assetId] ?? []).map(({ price, date }) => ({
       price,
       // TODO(0xdef1cafe): find best primitive to return/store this
-      date: new Date(Number(date) * 1000).toISOString()
+      date: date
     }))
 )
```

Comment:
hey @chxash - thanks for the PR. testing this locally it looks like this fixes things, so if that's the case, can you please go ahead and remove this whole `map` function, as it's not actually doing anything any longer?

Reply:
hi @0xdef1cafe, you are right :) I removed it